src/EnergyPlus/DualDuct.hh

```diff
@@ -219,6 +219,8 @@ namespace DualDuct {
         // Beginning of Update subroutines for the Damper Module
         // *****************************************************************************
 
+        void CalcOutdoorAirVolumeFlowRate();
+
```

Comment:
This change did not change the calculation instead moved the equation to a new member function. Any existing issues in dual duct are not addressed in this fix.